content/participants/ttjc@TarangRanpara/index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Tarang Ranpara
+type: "participant"
+name: "Tarang J Ranpara"
+portfolio: ""
+github-link: "https://github.com/TarangRanpara"
+linkedin-link: "https://www.linkedin.com/in/tarangranpara/"
+looking-for: "job"
+description: "Curious CS Student who is in love with every area of CS, and building quick prototypes!"
+---
+
+Hi! I am a Tarang Ranpara, let me walk you through my intro, projects and blogs.
```

Comment:
Hey @TarangRanpara i found a small typo here. 
* I am a Tarang ...

Reply:
Thanks @khusharth, I'll update it!